src/commands/scanner/run.ts

```diff
@@ -54,21 +54,19 @@ export default class Run extends ScannerRunCommand {
 			summary: getMessage(BundleName.Run, 'flags.targetSummary'),
 			description: getMessage(BundleName.Run, 'flags.targetDescription'),
 			delimiter: ',',
+			default: '.',
```

Comment:
We can't actually set the default because we need a way of determining if the user did or did not provide a value in order to determine whether or not to inform the user of the default that we set in the display. So although the default is indeed "."... we need to set it after seeing whether or not the provided value is null or not.